dpl-docs/views/ddox.layout.dt

```diff
@@ -5,6 +5,7 @@ block title
 	block ddox.title
 
 block navigation
+	include ddox.inc.utils
```

Comment:
`heading` has been moved in this auxiliary file: https://github.com/rejectedsoftware/ddox/blob/8ceb3815552765630e9fa0dece63f35837d416ee/views/ddox.inc.utils.dt